AutonomousDemo/Program.cs

```diff
@@ -10,9 +10,10 @@ class Program
     {
         static void Main(string[] arg)
```

Comment:
You should probably pull this program into a separate project and change the rest of this to be a library referenced by that program.